cmd/install/assets/hypershift_operator.go

```diff
@@ -1248,3 +1270,58 @@ func (o HyperShiftReaderClusterRoleBinding) Build() *rbacv1.ClusterRoleBinding {
 	}
 	return binding
 }
+
+type HyperShiftValidatingWebhookConfiguration struct {
+	Namespace *corev1.Namespace
+}
+
+func (o HyperShiftValidatingWebhookConfiguration) Build() *admissionregistrationv1.ValidatingWebhookConfiguration {
+	scope := admissionregistrationv1.NamespacedScope
+	path := "/validate-hypershift-openshift-io-v1alpha1-hostedcluster"
+	sideEffects := admissionregistrationv1.SideEffectClassNone
+	timeout := int32(10)
+	validatingWebhookConfiguration := &admissionregistrationv1.ValidatingWebhookConfiguration{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "ValidatingWebhookConfiguration",
+			APIVersion: admissionregistrationv1.SchemeGroupVersion.String(),
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: o.Namespace.Name,
+			Name:      "hypershift.openshift.io",
+			Annotations: map[string]string{
+				"service.beta.openshift.io/inject-cabundle": "true",
```

Comment:
fwiw this will prevent vanilla management from running webhook, which might be acceptable.